sass/pages/new-account/_new_account.scss

```diff
@@ -0,0 +1,90 @@
+body{
+	background-color: $lightGrey;
+}
+.new-div{
+	background-color: $lightGrey;
+
+	.new-form{
+		background-color: $lightGrey !important; 
+		width: 30%;
+		margin-right: auto;
+		margin-left: auto;
+		
+		.new-h2{
+			height: 5.14%;
+			color: #757779;
+			font-family:  Muli-ExtraLight;
+			font-size: 14px;
+			font-weight: 300;
+			line-height: 27px;
+			text-align: center;
+			margin: 0;
+		}
+
+		.new-h1{
+			height: 5.14%;
+			color: #757779;
+			font-family: Muli-Light;
+			font-size: 28px;
+			font-weight: 300;
+			line-height: 35px;
+			margin-bottom: 50px;
+			text-align: center;
+			margin: 0;
+		}
+
+		.new-inputs{
```

Comment:
I don't understand why you always use so many properties, I'm not sure you need many of them, take a better look at your code, try to use just the necessary. For example, if you define a font-family in the parent, all its children will use the same font, try to have things like that in mind.